Src/ILGPU/Backends/OpenCL/CLCodeGenerator.Emitter.cs

```diff
@@ -456,9 +465,15 @@ public void AppendConstant(float value)
                 }
                 else
                 {
-                    stringBuilder.Append(
-                        value.ToString(CultureInfo.InvariantCulture));
-                    if (value % 1.0f == 0.0f)
+                    // In C#, the floating point value "1.0f" can be shortened to "1f". However,
```

Comment:
nit: same here 🔢

Reply:
Done

Reply:
nit: please reformat to not exceed 90 chars per line

Reply:
Done